aspen/http/mapping.py

```diff
@@ -17,36 +17,42 @@ class Mapping(dict):
     clobbers to the list, while subscript access returns the last item. Think
     about it.
 
+    WARNING: this isn't thread-safe
```

Comment:
We're okay with it because aspen doesn't claim to be thread-safe, but I think it's a good practice to warn people when we know that a piece of our code isn't thread-safe.